samples/test/automlVisionObjectDetectionDataset.v1beta1.js

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
I don't see a change to `samples/package.json` - can you make sure `--recursive` is set on the `mocha` call?  Otherwise these tests won't get run 🙃

Reply:
fixed in the latest commit